onsite/phosphors.py

```
@@ -39,38 +733,52 @@ def get_occurrence_probability(exp_spectrum: MSSpectrum, tolerance_da: float) ->
 
-    # Formula: p = (NumPeaks * 2 * Tolerance) / Range
-    # The '2 * tolerance' represents the total window width around a peak.
-    occurrence_prob = (num_peaks * 2.0 * tolerance_da) / mz_range
+        # Ensure probability is within reasonable bounds
+        min_p = max(MIN_PROBABILITY, 0.01)  # Set minimum probability to 0.01
+        max_p = 0.5  # Set maximum probability to 0.5
+        occurrence_prob = max(min_p, min(occurrence_prob, max_p))
 
-    # Clamp probability between a minimum value and 1.0
-    return max(MIN_PROBABILITY, min(occurrence_prob, 1.0 - MIN_PROBABILITY)) # Prevent p=0 or p=1 for logs
+        print(f"Occurrence probability calculation:")
```

Ruff (0.14.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)